docs/api/en/math/Matrix4.html

```diff
@@ -315,7 +315,7 @@ <h3>
 			[method:this lookAt]( [param:Vector3 eye], [param:Vector3 target], [param:Vector3 up] )
 		</h3>
 		<p>
-			Constructs a rotation matrix, looking from [page:Vector3 eye] towards
+			Defines the rotation portion of a transformation matrix, looking from [page:Vector3 eye] towards
```

Comment:
How about this:

>Sets the rotation component of the transformation matrix, looking from [page:Vector3 eye] towards [page:Vector3 target], and oriented by the up-direction [page:Vector3 up].

In three.js, objects -- not matrices -- "look at". This is a method from the very early days of three.js, so it remains...

It probably should not be public.

It is just computing a 3x3 rotation matrix. I don't think `Matrix3` existed then.